Source/WebKit/Shared/EditorState.h

```diff
@@ -88,12 +88,12 @@ struct EditorState {
 #endif
 
     struct PostLayoutData {
-        uint32_t typingAttributes { AttributeNone };
+        OptionSet<TypingAttribute> typingAttributes { TypingAttribute::AttributeNone };
```

Comment:
I think this should work? (via template type deduction)

```suggestion
        OptionSet typingAttributes { TypingAttribute::AttributeNone };
```